core/types/src/transaction.rs

```diff
@@ -498,6 +501,17 @@ pub enum UpdateMethod {
     UpdateContentRegistry { updates: Vec<ContentUpdate> },
     /// Increment the node nonce.
     IncrementNonce {},
+    // Todo: In the future we should use a different type
+    // instead of `Job` that doesn't expose fields
+    // used for internal purposes, such as `assignee`.
+    /// Add new jobs to the jobs table and assign them to nodes.
+    AddJobs { jobs: Vec<Job> },
```

Comment:
I think we should define a max number of elements for the vecs and hashmaps of these three transaction types, and enforce it here https://github.com/fleek-network/lightning/blob/77964c4df5b8d0de9392a41dd57a22a1cd67209b/core/consensus/src/validator.rs#L55

Reply:
Sure, what do you think the max value should be?

Reply:
JobUpdates transactions should only be submitted by the node itself each time some work is done and anyone else should not be submitting these transactions manually. Is it ok to exclude this type of update from this validation?